pyproject.toml

```diff
@@ -52,12 +54,12 @@ build-backend = "setuptools.build_meta"
 include-package-data = true
 
 [tool.setuptools.packages.find]
-include = ["mesospim-stitcher*"]
+include = ["brainglobe_stitch*"]
 exclude = ["tests", "docs*"]
 
 
 [tool.pytest.ini_options]
-addopts = "--cov=mesospim-stitcher"
+addopts = "--cov=brainglobe_stitch"
```

Comment:
Should this be a hyphen or underscore?

Reply:
Based on bg-atlasapi, and brainrender-napari they should be underscores